django_fido/views.py

```diff
@@ -90,7 +90,7 @@ def create_fido2_request(self) -> Tuple[Dict, Dict]:
         """
         pass
 
-    def get(self, request: HttpRequest) -> JsonResponse:
+    def post(self, request: HttpRequest) -> JsonResponse:
```

Comment:
I don't quite like the change of the API here.

Reply:
We agreed on changing the API back to GET. Updated.